ledger-service/http-json-cli/src/main/scala/com/daml/http/OptionParser.scala

```diff
@@ -0,0 +1,169 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.http
+
+import java.nio.file.Paths
+
+import com.daml.ledger.api.tls.TlsConfigurationCli
+import org.slf4j.LoggerFactory
+import scopt.{Read, RenderingMode}
+
+import scala.concurrent.duration.{Duration, FiniteDuration}
+import scala.util.Try
+
+object OptionParser {
+
+  private val logger = LoggerFactory.getLogger(classOf[OptionParser])
+
+}
```

Comment:
I did this on a rush, rather than using raw Slf4j we probably want to use the same approach as in the rest of the HTTP JSON codebase and use Scala Logging instead.

Reply:
Address https://github.com/digital-asset/daml/pull/9219/commits/25a62a77e6b4e11ceaa4870aa3a134430f8348c7